shared/src/models/utils/alarm-group-vehicle.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNumber, IsUUID } from 'class-validator';
+import { uuid, UUID, uuidValidationOptions } from '../../utils';
+import { getCreate } from './get-create';
+
+export class AlarmGroupVehicle {
+    @IsUUID(4, uuidValidationOptions)
+    public readonly id: UUID = uuid();
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly vehicleTemplateId: UUID;
+
+    @IsNumber()
```

Comment:
```suggestion
    @IsNumber()
    @Min(0)
```

or even

```suggestion
    @IsNumber()
    @IsPositive()
```

Reply:
I think there could be a use case where you want vehicles to be there immediatly (depending on the goal of the exercise). Therefore I just used Min(0)